src/express-middleware/sitemap.js

```diff
@@ -1,5 +1,5 @@
 import express from 'express';
-import { generateSitemap } from '@plone/volto/helpers';
+import { generateSitemap } from '@plone/volto/helpers/Sitemap/Sitemap';
```

Comment:
I removed this helper module from helpers/index.js because it's for the server side only and uses the node `zlib` module.